app/executor.php

```diff
@@ -0,0 +1,1372 @@
+use Utopia\Database\Database;
+use Utopia\Database\Validator\Authorization;
+use Utopia\Database\Validator\UID;
+use Appwrite\Event\Event;
+use Appwrite\Utopia\Response\Model\Execution;
+use Appwrite\Messaging\Adapter\Realtime;
+use Appwrite\Stats\Stats;
+use Utopia\App;
+use Utopia\Swoole\Request;
+use Appwrite\Utopia\Response;
+use Utopia\CLI\Console;
+use Swoole\Process;
+use Swoole\Http\Server;
+use Swoole\Http\Request as SwooleRequest;
+use Swoole\Http\Response as SwooleResponse;
+use Utopia\Orchestration\Orchestration;
+use Utopia\Database\Adapter\MariaDB;
+use Utopia\Cache\Adapter\Redis as RedisCache;
+use Utopia\Config\Config;
+use Utopia\Validator\ArrayList;
+use Utopia\Validator\JSON;
+use Utopia\Validator\Text;
+use Cron\CronExpression;
+use Utopia\Storage\Device\Local;
+use Utopia\Storage\Storage;
+use Swoole\Coroutine as Co;
+use Utopia\Cache\Cache;
+use Utopia\Database\Query;
+use Utopia\Orchestration\Adapter\DockerCLI;
+use Utopia\Validator\Boolean;
+
+require_once __DIR__ . '/init.php';
+
+$dockerUser = App::getEnv('DOCKERHUB_PULL_USERNAME', null);
+$dockerPass = App::getEnv('DOCKERHUB_PULL_PASSWORD', null);
+$dockerEmail = App::getEnv('DOCKERHUB_PULL_EMAIL', null);
+$orchestration = new Orchestration(new DockerCLI($dockerUser, $dockerPass));
+
+$runtimes = Config::getParam('runtimes');
+
+Swoole\Runtime::enableCoroutine(true, SWOOLE_HOOK_ALL);
+
+// Warmup: make sure images are ready to run fast 🚀
+Co\run(function () use ($runtimes, $orchestration) {
+    foreach ($runtimes as $runtime) {
+        go(function () use ($runtime, $orchestration) {
+            Console::info('Warming up ' . $runtime['name'] . ' ' . $runtime['version'] . ' environment...');
+
+            $response = $orchestration->pull($runtime['image']);
+
+            if ($response) {
+                Console::success("Successfully Warmed up {$runtime['name']} {$runtime['version']}!");
+            } else {
+                Console::error("Failed to Warmup {$runtime['name']} {$runtime['version']}!");
+            }
+        });
+    }
+});
+
+/**
+ * List function servers
+ */
+$executionStart = \microtime(true);
+
+$response = $orchestration->list(['label' => 'appwrite-type=function']);
+
+$activeFunctions = new Swoole\Table(1024);
+$activeFunctions->column('id', Swoole\Table::TYPE_STRING, 512);
+$activeFunctions->column('name', Swoole\Table::TYPE_STRING, 512);
+$activeFunctions->column('status', Swoole\Table::TYPE_STRING, 512);
+$activeFunctions->column('key', Swoole\Table::TYPE_STRING, 4096);
+$activeFunctions->create();
+
+
+foreach ($response as $value) {
+    $activeFunctions->set($value->getName(), [
+        'id' => $value->getId(),
+        'name' => $value->getName(),
+        'status' => $value->getStatus(),
+        'private-key' => ''
+    ]);
+}
+
+$executionEnd = \microtime(true);
+
+Console::info(count($activeFunctions) . ' functions listed in ' . ($executionEnd - $executionStart) . ' seconds');
+
+App::post('/v1/execute') // Define Route
```

Comment:
IMO we should have a description even if we don't use it in swagger. It helps devs understand what the code is for.